java/fury-core/src/main/java/org/apache/fury/serializer/ReplaceResolveSerializer.java

```diff
@@ -44,7 +44,7 @@
  * will skip classname writing if object returned by `writeReplace` is different from current class.
  */
 @SuppressWarnings({"unchecked", "rawtypes"})
-public class ReplaceResolveSerializer extends Serializer {
+public class ReplaceResolveSerializer extends AbstractObjectSerializer {
```

Comment:
Since ReplaceResolveSerializer copy by copy returned object of `writePlace` method, it's unnecessary to extends  AbstractObjectSerializer

Reply:
> Since ReplaceResolveSerializer copy by copy returned object of `writePlace` method, it's unnecessary to extends AbstractObjectSerializer

okay